frontend/app/routes/public/apply/$id/adult-child/confirmation.tsx

```diff
@@ -66,11 +66,10 @@ export async function loader({ context: { configProvider, serviceProvider, sessi
   const homeProvinceTerritoryStateAbbr = state.contactInformation.homeProvince ? serviceProvider.getProvinceTerritoryStateService().findById(state.contactInformation.homeProvince)?.abbr : undefined;
 
   // Getting Country by Id
-  const countryMailing = serviceProvider.getCountryService().findById(state.contactInformation.mailingCountry);
-  invariant(countryMailing, `Unexpected mailing address country: ${state.contactInformation.mailingCountry}`);
```

Comment:
Any reason why this `invariant` check was removed? The same invariant check for home address country below has been left in.

(same comment for `review-information` and other `confirmation` routes in this PR)

Reply:
Because `state.contactInformation.mailingCountry` type is `string` compared to `state.contactInformation.homeCountry` that is `string` or `undefined`. `getCountryById` will throw a `CountryNotFoundException` when not found instead of returning `null`.